pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks-pac.yaml

```diff
@@ -0,0 +1,234 @@
+---
```

Comment:
why this leading `---` ?

Reply:
yaml notation

Reply:
Oh I see it is also in orig file. Weird.

Reply:
that is used for multi doc yamls, here we have single resource defined

Reply:
The thing is: some parser refuse to parse multi doc yaml files

Reply:
hmmm.. I see.. yea.. we can remove it.. but that *pac file is just a sole copy of the original one, except for the ClusterTask to Task change.. .that's on upstream as well https://raw.githubusercontent.com/knative/func/main/pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks.yaml

Reply:
I think it's fine now.